marlowe-playground-server/contracts/EscrowWithCollateral.hs

```diff
@@ -0,0 +1,106 @@
+{-# LANGUAGE OverloadedStrings #-}
+module EscrowWithCollateral where
+
+import           Language.Marlowe
+
+main :: IO ()
+main = print . pretty $ contract
+
+
+{- What does the vanilla contract look like?
+  - if Alice and Bob choose
+      - and agree: do it
+      - and disagree: Carol decides
+  - Carol also decides if timeout after one choice has been made;
+  - refund if no choices are made.
+-}
+
+contract :: Contract
+contract = When [Case (Deposit "bob" "bob" ada bobCollateral) contract]
+                10
+                Close
+
+collateralAlice :: Contract
+collateralAlice = When [Case (Deposit "alice" "alice" ada aliceCollateral) escrow]
+                10
+                Close
+
```

Comment:
It also seems like there is one collateral missing here. I looked at the JS version and looks fine, but this one looks incomplete

Reply:
will fix